src/index.css

```diff
@@ -0,0 +1,71 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 20px;
+  box-sizing: border-box;
+  background: #f3f2f2;
+}
+
+.main-container {
+  background-color: #fff;
+  border-radius: 5px;
+  margin: 10% 10%;
+  box-shadow: 3px 3px 3px 3px gray;
+  padding: 3%;
+}
+
+h1 {
+  font-size: 22px;
+  font-weight: normal;
+}
+
+input {
+  border: 0;
+  outline: 0;
+  padding: 0.2rem;
+}
+
+hr {
+  border: 0.1px solid #f3f2f2;
+  margin: 0;
+}
+
+i {
+  cursor: pointer;
+  color: gray;
+}
+
+.my-button {
+  width: 100%;
+  height: 100%;
+}
+
+.my-button:hover {
+  text-decoration: underline;
+  color: #000;
+}
+
+.head {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 3%;
+  padding-right: 3%;
+}
```

Comment:
Hi, styling for this element `h1` has already been added at line 17-20.

Styling has been done once using `h1` tag name above and again using class selector here. Please check if these two selectors can be combined.